tools/dct: Port DrvGen to Python 3

`tools/dct/DrvGen.py` is invoked through `scripts/drvgen/drvgen.mk` when
building the MTK DTS/cust.dtsi files. It used Python 2 syntax in five
spots, all purely syntactic.

Nixpkgs has dropped Python 2 entirely, so convert these rather than
pinning a dead interpreter. No behavioural change intended.

diff -urp a/tools/dct/DrvGen.py b/tools/dct/DrvGen.py
--- a/tools/dct/DrvGen.py	2026-10-03 00:49:08.218795220 +1000
+++ b/tools/dct/DrvGen.py	2026-10-03 00:49:43.791672435 +1000
@@ -37,7 +37,7 @@ from utility.util import LogLevel
 from utility.util import log
 
 def usage():
-    print '''
+    print('''
 usage: DrvGen [dws_path] [file_path] [log_path] [paras]...
 
 options and arguments:
@@ -46,7 +46,7 @@ dws_path    :    dws file path
 file_path   :    where you want to put generated files
 log_path    :    where to store the log files
 paras        :    parameter for generate wanted file
-'''
+''')
 
 def is_oldDws(path, gen_spec):
     if not os.path.exists(path):
@@ -55,7 +55,7 @@ def is_oldDws(path, gen_spec):
 
     try:
         root = xml.dom.minidom.parse(dws_path)
-    except Exception, e:
+    except Exception as e:
         log(LogLevel.warn, '%s is not xml format, try to use old DCT!' %(dws_path))
         if len(gen_spec) == 0:
             log(LogLevel.warn, 'Please use old DCT UI to gen all files!')
diff -urp a/tools/dct/utility/util.py b/tools/dct/utility/util.py
--- a/tools/dct/utility/util.py	2026-10-03 00:49:08.220795213 +1000
+++ b/tools/dct/utility/util.py	2026-10-03 00:49:43.791672435 +1000
@@ -27,11 +27,11 @@ class LogLevel:
 
 def log(level, msg):
     if level == LogLevel.info:
-        print LEVEL_INFO + msg
+        print(LEVEL_INFO + msg)
     elif level == LogLevel.warn:
-        print LEVEL_WARN + msg
+        print(LEVEL_WARN + msg)
     elif level == LogLevel.error:
-        print LEVEL_ERROR + msg
+        print(LEVEL_ERROR + msg)
 
 def compare(value):
     lst = re.findall(r'\d+', value)
